client/components/login/index.css

```diff
@@ -52,4 +52,7 @@ div.remember-me {
     height: 25px;
     bottom: -6px;
   }
+  .Toastify__toast-container {
+    height: 5vw;
```

Comment:
are you sure that `5vw` is a value old browsers would support?

Reply:
Yes, it supports. I tested it on emulator.

Reply:
I tested it on samsung browser and it doesn't support it,

Can you import the CSS only if polyfill is loaded and use max-height with a fixed pixel instead? I would also try adding a white background and dark gray text color to the toast to make sure it shows with color (I see a transparent box instead), as long as this CSS is loaded only from the polyfill file we should be fine.